src/Features/Core/Portable/Completion/SharedSyntaxContextsWithSpeculativeModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.Extensions.ContextQuery;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Completion
+{
+    internal sealed class SharedSyntaxContextsWithSpeculativeModel
+    {
+        public Document Document { get; }
+        public int Position { get; }
```

Comment:
```suggestion
        public Document Document { get; }
        public int Position { get; }
```

Do these need to be public?